content/donate/index.af.md

```diff
@@ -51,16 +57,6 @@ Klik op u voorkeurbetaalmetode-ikoon hier onder:
    - Openbare vervoer
    - … en [vele ander][github issues] funksies wat u wil hê en waarvan u hou
 
-## Hoe om te skenk?
-
-Klik op u voorkeurbetaalmetode-ikoon hier onder:
-
-{{ donate_buttons() }}
```

Comment:
Please check the English version (and a few other langs that are not yet synchronized with Weblate). We display donate buttons twice:
1. If someone already wants to donate without reading the details, the upper donate block is immediately visible on a mobile phone screen.
2. If someone is not sure, then there is another donate block below, after reading the motivating text (it will be improved later).

Removal of the second block may not be convenient, think of mobile users first.

Reply:
1. I checked and updated all the missing translations manually. I will try to make them sync with Weblate as discussed.
2. The text at the top is not huge and it won't prevent anyone from seeing the donation buttons immediately.
3. Repetitive donations buttons are not good UX imo, because it looks too forceful and insisting instead of convincing.